flash/data/data_pipeline.py

```diff
@@ -0,0 +1,517 @@
+import functools
+import os
+import weakref
+from functools import partial, wraps
+from typing import Any, Callable, Iterable, Mapping, Optional, Sequence, Tuple, Type, TYPE_CHECKING, Union
+
+from pytorch_lightning.trainer.connectors.data_connector import _PatchDataLoader
```

Comment:
```suggestion
from pytorch_lightning.trainer.connectors.data_connector import PatchDataLoader
```
what us the motivation for having protected classes?

Reply:
This was already protected in lightning, we just need it here for some checks. We didn't add this and since it usually is a lightning internal class that is never exposed to the user we kept it protected